linchpin/cli/__init__.py

```diff
@@ -187,10 +187,14 @@ def _write_distilled_context(self, run_data):
         dist_roles = self.get_cfg('distiller')
 
         resources_path = self.get_evar('resources_folder')
-        context_path = '{0}/{1}'.format(self.workspace, resources_path)
-        if not os.path.exists(context_path):
-            os.makedirs(context_path)
-        context_file = '{0}/{1}'.format(context_path, 'linchpin.distilled')
+        if os.path.isabs(resources_path):
+            context_file = '{0}/{1}'.format(resources_path,
+                                            'linchpin.distilled')
+        else:
+            context_path = '{0}/{1}'.format(self.workspace, resources_path)
+            if not os.path.exists(context_path):
+                os.makedirs(context_path)
```

Comment:
It's possible this `resources_path` may not exist. Might be good to make sure it's created before putting anything into that path as well.